frontend/src/tests/assignment-tests.js

```diff
@@ -111,24 +113,36 @@ export function assignmentsTests(api) {
         checkPropTypes(assignmentPropTypes, updatedAssignment);
         expect(updatedAssignment.id).not.toBeNull();
         expect(updatedAssignment.id).toEqual(assignment.id);
+    });
+
+    it("assignments created with null start/end_date inherit the start/end_date from the parent position", async () => {
 
-        // fetch all assignments and make sure the newly added
-        // assignment data is there
-        const { payload: withUpdatedAssignment } = await apiGET(
-            `/admin/sessions/${session.id}/assignments`
-        );
 
-        expect(withUpdatedAssignment.map((x) => x.id)).toContain(
-            updatedAssignment.id
+        const newPositionData = {
+            position_code: "CSC200F",
+            position_title: "Basic Computer Science",
+            hours_per_assignment: 70,
+            start_date: "2019-09-05T00:00:00.000Z",
+            end_date: "2020-01-05T00:00:00.000Z",
+            contract_template_id: contractTemplate.id,
+        };
+
+        var resp = await apiPOST(
+            `/admin/sessions/${session.id}/positions`,
+            newPositionData
         );
```

Comment:
I believe a check here to see if the position is successfully posted in needed, in order to ensure the assignment goes to a valid position.